extensions/gitlab/src/components/event.tsx

```diff
@@ -72,32 +84,41 @@ export function EventListItem(props: { event: Event }) {
         const an = capitalizeFirstLetter(ev.action_name);
         const pd = ev.push_data;
         if (pd) {
+          let iconColor: ColorLike | undefined;
+          switch (ev.action_name) {
+            case "pushed new":
+              {
+                iconColor = Color.Purple;
+              }
+              break;
+            case "pushed to":
+              {
+                iconColor = Color.Green;
+              }
+              break;
+            case "deleted":
+              {
+                iconColor = Color.Red;
+              }
+              break;
+          }
+          let iconSource: ImageSource | undefined;
           if (pd.ref_type === "branch") {
             title = `${an} branch ${pd.ref}`;
-            switch (ev.action_name) {
-              case "pushed new":
-                {
-                  icon = { source: GitLabIcons.branches, tintColor: Color.Purple };
-                }
-                break;
-              case "pushed to":
-                {
-                  icon = { source: GitLabIcons.branches, tintColor: Color.Green };
-                }
-                break;
-              case "deleted":
-                {
-                  icon = { source: GitLabIcons.branches, tintColor: Color.Red };
-                }
-                break;
-            }
+            iconSource = GitLabIcons.branches;
+          } else if (pd.ref_type === "tag") {
+            title = `${an} tag ${pd.ref}`;
+            iconSource = Icon.Pin; // todo: use gitlab tag icon
```

Comment:
Change the icon first?

Reply:
Could you give some advice?
I've found this one (https://gitlab.com/gitlab-org/gitlab-svgs/blob/main/sprite_icons/tag.svg) but I couldn't download it as PNG and I also don't really like it. What do you think?

Reply:
https://svgtopng.com could be an option. Just make sure that the png is big enough.

Reply:
But I'm also fine with the Raycast icon.